src/transports/https.js

```diff
@@ -29,17 +30,21 @@ class HTTPSStream extends Writable {
     * @param {string} [options.httpsAgent] - Your own custom https.Agent. We use agents to maintain connection pools and keep the connections alive. This avoids the initial connection overhead every time we want to communicate with Timber. See https.Agent for options.
   */
   constructor(apiKey, {
-      flushInterval = 2500,
-      httpsAgent,
-      httpsClient
-    } = {})
-  {
-    super({
-      objectMode: true,
-      highWaterMark: 5000
-    });
+    flushInterval = 2500,
```

Comment:
To example on this, we're using keep alive connections, so the general connection overhead is minimized. We should flush the buffer at least once a second, imo.